### DISCUSSION TITLE
ButtonLink spinner

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#795] [Proposals]
This designs is applying a buttonLink with a spinner using inline.

Should be allowed by Mística?
The rest of buttons has loading state.

![image](https://user-images.githubusercontent.com/6722153/191981675-6e86d69e-e900-4bee-a158-f4dda8de214e.png)

**Comment by AnaMontes11** [comment]
I would add the loading state to maintain consistency and to avoid this type of cases.

**Comment by yceballost** [comment]
issue created
https://github.com/Telefonica/mistica-design/issues/798
